### ISSUE TITLE
Support text inputs

### ISSUE THREAD

**Issue by qkdreyer** [mengxiong10/vue2-datepicker#141]
Would it be possible to support text input ?
I could also try to implement it if you could guide me a little

This plugin has implemented it : http://g.recordit.co/9AeBccW1EQ.gif
Link : http://blog.w3cub.com/vue2-calendar/

**Comment by mengxiong10** [comment]
v2.x supports text input by default.

**Comment by qkdreyer** [comment]
I didn't manage to use it on your demo page : http://recordit.co/ll71gTGdzj
This is using https://unpkg.com/vue2-datepicker@2.4.0/lib/index.js

**Comment by mengxiong10** [comment]
Yes, because it set the `not-before`. so you types a invalid date, it don't change.

**Comment by qkdreyer** [comment]
Indeed, thanks.
Would it be possible to update the popup date while typing ?

**Comment by mengxiong10** [comment]
You can easily implement it by modifying the method.
```js
import DatePicker from 'vue2-datepicker'

const handleInput = DatePicker.methods.handleInput

DatePicker.methods.handleInput = function (event) {
  handleInput.call(this, event)
  this.handleChange(event)
  this.showPopup()
}

```
